ExampleMod/Content/Tiles/ExampleChunkLarge.cs

```diff
@@ -0,0 +1,63 @@
+using ExampleMod.Content.Items;
+using Microsoft.Xna.Framework;
+using Terraria;
+using Terraria.DataStructures;
+using Terraria.GameContent;
+using Terraria.ID;
+using Terraria.ModLoader;
+using Terraria.ObjectData;
+
+namespace ExampleMod.Content.Tiles
+{
+	public abstract class ExampleChunkLargeBase : ModTile {
+		public override void SetStaticDefaults() {
```

Comment:
as per [Style guide](https://github.com/tModLoader/tModLoader/wiki/tModLoader-Style-Guide), opening braces after classes should be on a new line. Same goes for all the other files in this PR, I won't highlight them separately.
Also, since it's ExampleMod, might be necessary to add a comment on the abstract classes that says something like "implementing classes further down the file"